PowerUp/src/Subsystems/DriveTrain.h

```diff
@@ -41,10 +41,10 @@ class DriveTrain: public frc::Subsystem {
 	constexpr static double lowGgearRatio = 15.0 / 1.0;
 	constexpr static double lowGgearEncoderRatio = lowGgearRatio * (12.0 / 30.0);
 	constexpr static double sensorUnitsPerRev = 128 * 4; // The TalonSRX counts 4 edges per encoder count, the quadrature encoder has 256 counts per revolution
-	constexpr static double nativeUnitsPerMeterLowGear = 1 / wheelCircumference * lowGgearEncoderRatio * sensorUnitsPerRev; // Approx. 19249
+	constexpr static double nativeUnitsPerMeterLowGear = 1 / wheelCircumference * lowGgearEncoderRatio * sensorUnitsPerRev; // Approx. 19249 for 256 count per rev, 9624 fir 128 count per rev
```

Comment:
Super minor, but there is a typo in the comment

Reply:
Cool, thanks